discovery-provider/src/queries/get_notifications.py

```diff
@@ -72,6 +72,38 @@
     bindparam("valid_types", expanding=True)
 )
 
+unread_notification_count_sql = text(
+    """
+--- Create Intervals of user seen
+SELECT
+    count(*)
+FROM (
+   select n.type, n.group_id
+   from
+       notification n
+  WHERE
+    ARRAY[:user_id] && n.user_ids AND
+    (:valid_types is NOT NULL AND n.type in :valid_types) AND
+    n.timestamp > COALESCE((
+        SELECT
+            seen_at
+        FROM
+            notification_seen
+        WHERE
+            user_id = :user_id
+        ORDER BY seen_at desc
+        LIMIT 1
+    ), '2016-01-01'::timestamp) AND 
```

Comment:
for my learning whats this 2016 date referring to??

Reply:
it's just a date really far in the past because comparison again null will return false - this is the backup in case there's no result from the query using coalesce